.golangci.yml

```diff
@@ -38,81 +55,69 @@ linters:
     - musttag
     - nakedret
     - nilerr
+    - noctx
+    - nolintlint
     - nosprintfhostport
     - perfsprint
     - prealloc
     - predeclared
     - promlinter
     - protogetter
     - reassign
+    - rowserrcheck
```

Comment:
Not sure if we need to enable this one?

Reply:
I'd say leave it enabled if we either add code which the linter may catch in the future.